vars/perfRegressionParallelPipeline.groovy

```diff
@@ -45,62 +54,116 @@ def call(Map pipelineParams) {
+                                                        set -xe
+                                                        env
+
+                                                        export SCT_CLUSTER_BACKEND=${params.backend}
+                                                        export SCT_REGION_NAME=${pipelineParams.aws_region}
+                                                        export SCT_CONFIG_FILES=${pipelineParams.test_config}
+
+                                                        if [[ ! -z "${params.scylla_ami_id}" ]] ; then
+                                                            export SCT_AMI_ID_DB_SCYLLA=${params.scylla_ami_id}
+                                                        elif [[ ! -z "${params.scylla_version}" ]] ; then
+                                                            export SCT_SCYLLA_VERSION=${params.scylla_version}
+                                                        elif [[ ! -z "${params.scylla_repo}" ]] ; then
+                                                            export SCT_SCYLLA_REPO=${params.scylla_repo}
+                                                        else
+                                                            echo "need to choose one of SCT_AMI_ID_DB_SCYLLA | SCT_SCYLLA_VERSION | SCT_SCYLLA_REPO"
+                                                            exit 1
+                                                        fi
+
+                                                        export SCT_POST_BEHAVIOR_DB_NODES="${params.post_behavior_db_nodes}"
+                                                        export SCT_POST_BEHAVIOR_LOADER_NODES="${params.post_behavior_loader_nodes}"
+                                                        export SCT_POST_BEHAVIOR_MONITOR_NODES="${params.post_behavior_monitor_nodes}"
+                                                        export SCT_INSTANCE_PROVISION=${pipelineParams.params.get('provision_type', '')}
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$GIT_BRANCH | sed -E 's+(origin/|origin/branch-)++')
+                                                        export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$SCT_AMI_ID_DB_SCYLLA_DESC | tr ._ - | cut -c1-8 )
+
+                                                        echo "start test ......."
+                                                        ./docker/env/hydra.sh run-test ${pipelineParams.test_name}.${current_sub_test} --backend ${params.backend}  --logdir /sct
+                                                        echo "end test ....."
+                                                        """
+                                                    }
+                                                }
+                                            }
+                                        }
+                                        stage('Clean resources') {
```

Comment:
@aleksbykov does this pipeline doesn't have collect logs step on purpose ?

Reply:
WIll add.

Reply:
Added